ci-operator/config/bentitopolis/operator-metering/bentitopolis-operator-metering-master.yaml

```
@@ -0,0 +1,70 @@
+base_images:
+  ansible-operator:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: ansible-operator
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    context_dir: .
+    dockerfile_path: Dockerfile.src
+images:
+- dockerfile_path: Dockerfile.reporting-operator.okd
+  from: base
+  to: metering-reporting-operator
+- dockerfile_path: Dockerfile.metering-ansible-operator
+  from: ansible-operator
+  to: metering-ansible-operator
+promotion:
+  disabled: true
+  name: "4.5"
+  namespace: ocp
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: make unit TEST_OUTPUT_DIR=/tmp/artifacts
+  container:
+    from: src
+- as: verify
+  commands: |
+    set -x
+    set -e
+    # (chance): Hacks to allow us write access to our source files since
+    # they're 664 in the image and owned by root:root, and we get a random UID
+    # in Openshift pods which means we don't have write permissions.
+
+    # create a copy of the source so we can modify the files.
+    export GOPATH=/tmp/go
+    mkdir -p $GOPATH/src/github.com/bentitopolis/
+    cp -r /go/src/github.com/bentitopolis/operator-metering/ $GOPATH/src/github.com/bentitopolis/operator-metering
+    cd $GOPATH/src/github.com/bentitopolis/operator-metering
+
+    # Run the actual tests that nothing in the changes after re-generating
+    # files. This ensures generated files aren't modified directly.
+    make verify
+  container:
+    from: src
+- as: metering-e2e-aws
```

[error: could not run steps: step metering-e2e-aws failed: could not wait for template instance to be ready: failed to create objects: rolebindings.authorization.openshift.io "metering-e2e-aws-image-puller" already exists
](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/7622/rehearse-7622-pull-ci-bentitopolis-operator-metering-master-metering-e2e-aws/15)

```suggestion
- as: bentitopolis-metering-e2e-aws
```

---

oh okay will try the change soon